server/functions.lua

```diff
@@ -430,6 +430,11 @@ function ClearInventory(source, filterItems)
     if not player.Offline then
         local logMessage = string.format('**%s (citizenid: %s | id: %s)** inventory cleared', GetPlayerName(source), player.PlayerData.citizenid, source)
         TriggerEvent('qb-log:server:CreateLog', 'playerinventory', 'ClearInventory', 'red', logMessage)
+        local ped = GetPlayerPed(source)
+        local weapon = GetSelectedPedWeapon(ped)
+        if weapon ~= GetHashKey('WEAPON_UNARMED') then
```

Comment:
Good change. It'd be better to use backticks rather than calculate the hash each time with the native.
```suggestion
        if weapon ~= `WEAPON_UNARMED` then
```

Reply:
@Qwerty1Verified 
Updated to use backticks to avoid unnecessary runtime hash calculations and improve efficiency.
Thanks.